skimage/transform/tests/test_radon_transform.py

```diff
@@ -261,9 +290,12 @@ def argmax_shape(a):
             argmax_shape(sinogram_circle_to_square))
 
 
-def test_sinogram_circle_to_square():
-    for size in (50, 51):
-        yield check_sinogram_circle_to_square, size
+sizes_for_test_sinogram_circle_to_square = (50, 51)
```

Comment:
I think that short lists of parameters could be put directly into `pytest.mark.parametrize` (without specifying separate variable). Here and above

Reply:
1) OK, I will fix it.
2) I'm not sure where the "above" is.

Reply:
@nelzas I meant these lines - https://github.com/scikit-image/scikit-image/pull/2824/files#diff-fc1cd516a866b1cdcf5ed18a3d972af0R71, https://github.com/scikit-image/scikit-image/pull/2824/files#diff-fc1cd516a866b1cdcf5ed18a3d972af0R74

Reply:
No objection to fix the first one, I'm more reluctant about the second one as I find it less readable - a product of various lists without clear meaning:
```python
@pytest.mark.parametrize("size, theta, circle",
                         itertools.product([16, 17],
                                           [0, 90],
                                           [False, True])
```

Reply:
@nelzas feel free to submit another PR with the refactoring of the current tests. :)
You've done a great job already, and I consider the original issue fixed, at this point.